src/xrm-mock-generator/xrm-mock-generator.ts

```diff
@@ -16,12 +16,13 @@ export class XrmMockGenerator {
   public static Ui: Ui = new Ui();
   public static WebApi: WebApi = new WebApi();
 
-  public static initialise(): XrmMock.XrmStaticMock {
+  public static initialise(entityName: string): XrmMock.XrmStaticMock {
```

Comment:
I'm guessing we should accept a new interface for the properties for initialise, which are all optional, and then default them if they are not provided.  This will break every's current call to initialise correct?

We are already defaulting an Id on line 24.  I vote we create a new interface that has the entity Id, and entity Name as optional fields, and then we default them to "{deadbeef-dead-beef-dead-beefdeadbeaf}" (so it's easily searchable and memorable) and "contact".  That way if the test really cares, they can set them, if not, let the default's rule.

Reply:
Good call. I can make that change.